File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list)
{
    int even = 0, odd = 0;
    vector<int> answer;

    for(auto& num : num_list) {
        if(num % 2 == 0) {
            even += 1;
        } else {
            odd += 1;
        }
    }
    answer.push_back(even);
    answer.push_back(odd);

    return answer;
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    even = []
    odd = []

    for x in num_list:
        if x % 2 == 0:
            even.append(x)
        else:
            odd.append(x)
    answer.append(len(even))
    answer.append(len(odd))

    return answer